packages/react/src/ReactServerContext.js

```diff
@@ -30,6 +30,13 @@ export function createServerContext<T: ServerContextJSONValue>(
   if (!enableServerContext) {
     throw new Error('Not implemented.');
   }
+  if (__DEV__) {
+    console.error(
+      'Server Context is deprecated and will soon be removed. ' +
+        'It was never documented and we have found it not to be useful ' +
+        'enough to warrant the downside it emposes on all apps.',
```

Comment:
```suggestion
        'enough to warrant the downside it imposes on all apps.',
```

Reply:
+1. but also nitpicking more: "the downside" → which downside?

maybe "…not be useful enough given the performance optimizations it prevents."

people love performance

Reply:
Didn’t really want to litigate in the logs. Maybe I opened for it but meh.